orga/mirage/config.js

```diff
@@ -189,6 +189,9 @@ export default function () {
     });
 
     prescriber.memberships = [membership];
+    if (code === 'ENGLISHGH01') {
```

Comment:
**Question** : A quoi correspond le code `ENGLISHGH01` ? 😄

Reply:
C'est un code de campagne à l’orthographe un peu bizarre en effet, mais c'est pour mieux le repérer sûrement 😉 … provenant vraisemblablement d'un copier-coller du code `ABCDEFGH01` situé dans le même fichier.

Qu'est-ce que tu suggèrerais @er-lim à la place ? Quelque chose comme ce qui suit ou autre ?

```suggestion
const ENGLISH_CAMPAIGN_CODE = 'ENGLISH01'
    if (code === ENGLISH_CAMPAIGN_CODE) {
```

Reply:
Ou peut-être que tu pensais à quelque chose de moins lourd ?

```suggestion
    if (code === 'ENGLISH_CAMPAIGN01') {
```

ou tout simplement ?

```suggestion
    if (code === 'ENGLISH01') {
```

Reply:
C'est ça, de créer une constante pour améliorer la lisibilité du fichier 😄

Reply:
@er-lim c'est juste un code campagne pour que l'appel api subbé par mirage renvoie un utilisateur avec la langue anglaise.  
C'est pas un code campagne mais le code d'une invitation, je suggère 
'ENGLISH_USER_INVITATION_CODE'
un peu long certes ^^ mais peut-être moins ambigu

Dans mirage config ça donnerait

```
 if (code === 'ENGLISH_USER_INVITATION_CODE') {
      prescriber.lang = 'en';
    }
```

Ps : on remarquera que le user d'une orga est un prescriber